server/lib/src/repl/proto.rs

```diff
@@ -6,6 +6,7 @@ use crate::prelude::*;
 use crate::schema::{SchemaReadTransaction, SchemaTransaction};
 use crate::valueset;
 use base64urlsafedata::Base64UrlSafeData;
+use kanidm_proto::internal::ImageValue;
```

Comment:
Could also be good to split the internal representation from the proto one in case we end up with multiple proto versions that our internal rep has to satisfy in future.

Reply:
Created `ReplImageV1`

Reply:
I think in future the repl proto and the db proto are really similar, since they both are just about serialisation. So I might end up deduplicating-merging them into a serialisable format crate.

Reply:
Seems sensible to me, I can make them the same in this instance?

Reply:
Yeah, lets do that for now. I'll have to commit to the repl proto cleanup before we stamp as good for production anyway.